ActionBuilder/package.json

```diff
@@ -26,22 +26,22 @@
     "lit-html": "1.2.1"
   },
   "peerDependencies": {
-    "@cjaas/common-components": "^2.1.1",
-    "@momentum-ui/core": "^19.10.0",
-    "@momentum-ui/icons": "^8.8.2",
-    "@momentum-ui/utils": "^6.2.12",
-    "@momentum-ui/web-components": "^2.2.0",
+    "@cjaas/common-components": "2.2.0-dev1",
+    "@momentum-ui/core": "19.15.3",
+    "@momentum-ui/icons": "8.20.1",
+    "@momentum-ui/utils": "6.2.15",
+    "@momentum-ui/web-components": "^2.4.0-dev2",
```

Comment:
We can move past the "dev" versions here, common comp 2.2.0 and momentum 2.4.0 are live!

Reply:
Great